lib/blaze/blaze.dart

```diff
@@ -326,7 +326,7 @@ class Blaze {
     _host = _host == _wsHost1 ? _wsHost2 : _wsHost1;
 
     try {
-      _disconnect();
+      _disconnect(false);
```

Comment:
Do not reset connected state to `ConnectedState.disconnected`, since it should be `reconnecting` state when doing a "http ping";